docs/assemblies/assembly_configuring-a-thorntail-application.adoc

```diff
@@ -23,6 +23,13 @@ include::../modules/proc_setting-system-properties-using-the-command-line.adoc[l
 
 include::../modules/proc_specifying-jdbc-drivers-for-hollow-jars.adoc[leveloffset=+2]
 
+[id='configuring-a-thorntail-application-using-environment-variables_{context}']
+== Environment Variables
+
+Using environment variables for configuring your application, or overriding values provided in YAML files, is advantageous for environments like Docker, Kubernetes, and OpenShift.
```

Comment:
For simplicity, I suggest you rephrase the sentence as:

```
Use environment variables to configure your application or override values stored in YAML files.
ifndef::product[]
This approach is useful in deployments like Docker, Kubernetes, or OpenShift.
endif::[]
```

I have restricted the information about the deployments to community docs only because in the product, we assume only OpenShift deployment AFAICT. If that's not the case, a similar sentence mentioning only OpenShift should be added for prod builds.

Reply:
I modified it, but with slightly different wording as the above made it seem like Docker, etc were the deployment as opposed to the environment